common/environment/environment.cpp

```diff
@@ -75,6 +75,7 @@ class CConstDropZoneServerInfoIterator : public CSimpleInterfaceOf<IConstDropZon
     Owned<CLocalEnvironment> constEnv;
     Owned<IPropertyTreeIterator> serverListIt;
     unsigned maxIndex = 0;
+    Owned<IPropertyTree> legacyServerList;
```

Comment:
not needed as member, getElements will link parent it's iterating.
So this could be moved into if (0 != dropZoneMachineName.length()) block in CConstDropZoneServerInfoIterator constructor.